### ISSUE TITLE
Adjust stroke margin

### ISSUE THREAD

**Issue by tchar** [hoang8f/android-segmented-control#21]
@hoang8f Hello. In relation to issue #20 about corners I was wondering if you would like to add support for border margin also (the stroke width in fact) and if someone wanted to change it.
For a user to change it it would require to tweak the code itself.

I have made some adjustments to the code, so now anyone can adjust the stroke width (or height) from the dimens file.

I also intend to implement xml attributes so devs can adjust each radiogroup's corner radius and border magin direclty from the xml layout (if they want) or use their dimens.xml to declare dimensions for border radius (which supported currently) and stroke width (which is not). So basically this will let someone to have different styles of radiogroups.
So basically the order will be:
Read the attributes. If app:border_radius, app:border_width is declared then use them.
Else use the values from the dimens.xml file.
Read the attributes. If app:border_radius, app:border_width is declared then use them.
Else use the values from the dimens.xml file.

Also I made some code polishing in updateBackground method.

If you are interested in any of those adjustments tell me to push the code to my forked repo so you can pull.

Thanks.
